src/sage/schemes/elliptic_curves/gal_reps_number_field.py

```diff
@@ -402,6 +402,369 @@ def reducible_primes(self):
 
         return [l for l in self.isogeny_bound() if self.E.isogenies_prime_degree(l)]
 
+    def is_borel(self, p):
+        r"""
+        Return ``True`` if the projective mod-`p` image is contained in a Borel subgroup.
+
+        INPUT:
+
+        - ``p`` (integer) -- a prime number
+
+        OUTPUT:
+
+        (boolean) ``True or ``False``.
+
+        EXAMPLES::
+
+            sage: from sage.schemes.elliptic_curves.gal_reps_number_field import GaloisRepresentation
+            sage: E = EllipticCurve('56b1')
+            sage: G = GaloisRepresentation(E)
+            sage: G.is_borel(2)
+            True
+            sage: E = EllipticCurve('50a1')
+            sage: G = GaloisRepresentation(E)
+            sage: G.is_borel(3)
+            True
+            sage: G.is_borel(5)
+            True
+            sage: E.has_cm()
+            False
+            sage: E = EllipticCurve('405d1')
+            sage: G = GaloisRepresentation(E)
+            sage: G.is_borel(7)
+            True
+            sage: E = EllipticCurve('121a1')
+            sage: G = GaloisRepresentation(E)
+            sage: G.is_borel(11)
```

Comment:
```suggestion
            sage: G.is_borel(11)  # long time
```

On a fast machine (i7-9700 CPU @ 3.00GHz):
```
Warning, slow doctest:
    G.is_borel(11)
Test ran for 1.57 s, check ran for 0.00 s
```

Reply:
OK (this one is longer since X_0(11) has genus >0)